libr/parse/parse.c

```diff
@@ -325,9 +327,15 @@ static int filter(RParse *p, RFlag *f, char *data, char *str, int len, bool big_
 									break;
 								}
 								memmove (ptr_left, ptr_esc, copied_len);
-								sprintf (ptr_left + copied_len, "%s%s",
-										ansi_found && ptr_right - ptr_end + 1 >= 4 ? "\x1b[0m" : "",
-										ptr_right + 1);
+								dptr_left = strcpy (ptr_left + copied_len,
+										ansi_found && ptr_right - ptr_end + 1 >= 4 ? "\x1b[0m" : "");
```

Comment:
so much scary pointer arithmetics here

Reply:
do not hardcode ansi escapecodes. this is not portable. use Color_RESET

Reply:
add parenthesis for this ternary conditional. or the >=, - and && operation may behave wrong in some compilers